compiler/rustc_lint/messages.ftl

```diff
@@ -532,8 +532,14 @@ lint_unknown_gated_lint =
 
 lint_unknown_lint =
     unknown lint: `{$name}`
-    .suggestion = did you mean
-    .help = did you mean: `{$replace}`
+    .suggestion = did you {$from_rustc ->
+        [true] `{$replace}`, a lint with the same name was found in `rustc` lints.
+        *[false] mean
```

Comment:
This is weird because emitter automatically uses verbose mode when the label is long.

Reply:
I think you can force the terse mode by replacing `#[suggestion(…)]` on `UnknownLintSuggestion::WithSpan` with `#[suggestion_short(…)]`.

Reply:
oh, good to know we have this attribute label, thanks for the tip!